torch_geometric/explain/algorithm/utils.py

```diff
@@ -63,6 +63,35 @@ def set_hetero_masks(
                     apply_sigmoid=apply_sigmoid,
                 )
 
+        elif (isinstance(module, (HANConv, HGTConv))):
+            # Skip if explicitly set for skipping
+            if (module.explain is False):
+                continue
+
+            loop_mask_dict = {
+                '__'.join(k): edge_index[0] != edge_index[1]
```

Comment:
We can use `EDGE_TYPE_STR_SPLIT` from `torch_geometric.typing` to replace `'__'`

Reply:
Added a commit that replaces all instances of `'__'` with `EDGE_TYPE_STR_SPLIT` in this file.